src/contact.html

```diff
@@ -28,8 +29,37 @@
         </nav>
     </header>
     <main class="content">
-        <!-- TODO -->
+        <div class="contact_text">
+            <p>
+                <b>Country:</b>
+                <br>
```

Comment:
@MiljkovicD Please solve spacings between elements with `css` instead of using the `<br>` tag.